tests/system/Autoloader/FileLocatorTest.php

```diff
@@ -119,4 +120,100 @@ public function testLocateFileReturnsEmptyWithBadNamespace()
 	}
 
 	//--------------------------------------------------------------------
+        
+        public function testSearchSimple()
+	{
+                $expected = rtrim(APPPATH,'/').'/Views/form.php';
```

Comment:
Not really necessary to remove the trailing slashes from out path constants, since they're all normalized with a trailing slash, but  no need to change it for these test.